client/flutter/android/app/src/main/kotlin/org/who/myhealth/MainActivity.kt

```diff
@@ -1,4 +1,4 @@
-package org.who.myhealth
+package org.who.WHOMyHealth
```

Comment:
This file needs to chnge directory to WHOMyHealth

Reply:
It works as is for me.

Reply:
With the cahgne it also works.

Reply:
think we should stick with std package naming